improver_tests/calibration/reliability_calibration/test_AggregateReliabilityCalibrationTables.py

```diff
@@ -112,6 +112,17 @@ def test_valid_bounds(self):
         plugin = Plugin()
         plugin._check_frt_coord([self.reliability_cube, self.different_frt])
 
+    def test_matching_bounds(self):
+        """Test that no exception is raised in the input cubes if a cube
+        contains matching bounds."""
+        reliability_cube = self.reliability_cube.copy()
```

Comment:
This test isn't currently testing what you think it is. You are modifying reliability_cube but passing in self.reliability_cube. I think it's probably best not to make a copy of self.reliability_cube.

Reply:
I've corrected the test.